### PR TITLE
[FUSEQE-6148] added list of all issues that affected the test run

### PR DESCRIPTION
**Pull request by mmuzikar**
//test: `@api-provider-delete`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Checkstyle verification failed with:
```
utilities/src/main/java/io/syndesis/qe/cucumber/
  MailFormatter.java:105:52: '{' is not preceded with whitespace. [WhitespaceAround]
utilities/src/main/java/io/syndesis/qe/cucumber/
  MailFormatter.java:108:32: '{' is not preceded with whitespace. [WhitespaceAround]
utilities/src/main/java/io/syndesis/qe/issue/
  SimpleIssue.java:19:27: '{' is not preceded with whitespace. [WhitespaceAround]
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
API Provider Integration | API Provider DELETE  - known issue
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
API Provider Integration | API Provider DELETE  - known issue
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Checkstyle verification failed with:
```
utilities/src/main/java/io/syndesis/qe/cucumber/
  MailFormatter.java:126:52: '{' is not preceded with whitespace. [WhitespaceAround]
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmuzikar** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmuzikar** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
API Provider Integration | API Provider DELETE  - known issue
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmuzikar** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
API Provider Integration | API Provider DELETE  - known issue
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Connection - CRUD | Create and delete 
Connection - CRUD | Kebab menu 
Connection - CRUD | Kebab menu edit and view 
API Provider Integration | API Provider DELETE  - known issue
Integration - DB to DB | Read and update operations 
Integration info | Check error for integration with the same name 
Integration info | Check error for integration which has same name after update 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmuzikar** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
API Provider Integration | API Provider DELETE  - known issue
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
API Provider Integration | API Provider DELETE  - known issue
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
API Provider Integration | API Provider DELETE  - known issue
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmuzikar** [comment]
Result: [https://fuse-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/syndesis-tests-PR/1608/artifact/syndesis-qe/results/ui/cucumber-mail/report.html](https://fuse-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/syndesis-tests-PR/1608/artifact/syndesis-qe/results/ui/cucumber-mail/report.html)

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: utilities/src/main/java/io/syndesis/qe/cucumber/MailFormatter.java [hunk 1]
```diff
@@ -108,6 +108,22 @@ private void onTestRunFinished(TestRunFinished t) {
             for (String line : results.stream().map(ScenarioResult::toString).collect(Collectors.toList())) {
                 out.write(line + " <br/>\n");
             }
+            out.write("<h2>All open issues: </h2>");
+            out.write("<ul>");
+            List<SimpleIssue> allOpenIssues = new ArrayList<>();
+            results.forEach(scenarioResult -> {
+                if (scenarioResult.getIssues() != null) {
+                    scenarioResult.getIssues().forEach(issue -> {
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/cucumber/MailFormatter.java]: you can use `allOpenIssues.addAll(...stream().filter(...).collect(to list)), but that is pretty much the same :dancer:

**Comment by avano** [comment]
@mmuzikar do you have an example report with this change? I think it will break something in the jenkins pipeline :grin:

**Comment by mmuzikar** [comment]
@avano minimal report: 
https://fuse-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/syndesis-tests-PR/1608/artifact/syndesis-qe/results/ui/cucumber-mail/report.html

**Comment by avano** [comment]
thx, I'll check that out

File: utilities/src/main/java/io/syndesis/qe/cucumber/MailFormatter.java (hunk 1 above)
Comment (mmuzikar) [reply utilities/src/main/java/io/syndesis/qe/cucumber/MailFormatter.java]: I did flatStream on results before but that caused issues on Jenkins so I went oldschool :space_invader:

**Comment by avano** [comment]
@mmuzikar can you please create a new file with the issues only? without any header and stuff? It would simplify my life :grin: Then I will just include it in the mail report if the file exists

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
API Provider Integration | API Provider DELETE  - known issue
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmuzikar** [comment]
@avano [https://fuse-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/syndesis-tests-PR/1632/artifact/syndesis-qe/results/ui/cucumber-mail/issues.html](https://fuse-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/syndesis-tests-PR/1632/artifact/syndesis-qe/results/ui/cucumber-mail/issues.html)

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
API Provider Integration | API Provider DELETE  - known issue
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**